docs/docs/references/users/README.md

```diff
@@ -1,5 +1,95 @@
+
+:::note The meanings of some following table columns
+
+- **Unique**: Ensures the _uniqueness_ of the values entered into an attribute of a DB table.
+- **Required**: Ensures that the values entered an attribute of a DB table can NOT be NULL.
+- **Editable**: Admin users can only edit the _editable_ attributes using Admin Console or Management API.
+- **Searchable**: Admin users can only search the keyword on the _searchable_ attributes using Admin Console or Management API.
+
+:::
+
+<br/>
+
+| Name                        | Type         | Description                                   | Unique | Required | Editable | Searchable |
+| --------------------------- | ------------ | --------------------------------------------- | ------ | -------- | -------- | ---------- |
+| id                          | text         | Unique identifier, auto-generate on creating  | ✅     | ✅       | ❌       | ❌         |
+| username                    | text         | Username for sign-in                          | ✅     | ❌       | ❌       | ✅         |
+| primary_email               | text         | Primary email                                 | ✅     | ❌       | ✅       | ✅         |
+| primary_phone               | text         | Primary phone number                          | ✅     | ❌       | ✅       | ✅         |
+| name                        | text         | Full name                                     | ❌     | ❌       | ✅       | ✅         |
+| avatar                      | text         | URL pointing to user's avatar image           | ❌     | ❌       | ✅       | ❌         |
+| role_names                  | string array | List of roles                                 | ❌     | ✅       | ❌       | ❌         |
+| [identities](#identities)   | object       | User info retrieved from social sign-in       | ❌     | ✅       | ❌       | ❌         |
+| [custom_data](#custom-data) | object       | Additional info in customizable attributes    | ❌     | ✅       | ✅       | ❌         |
+| application_id              | text         | Application ID that the user first registered | ❌     | ✅       | ❌       | ❌         |
+| last_sign_in_at             | date time    | Timestamp when the user signed in last time   | ❌     | ✅       | ❌       | ❌         |
+
+Two other password-related attributes are invisible on the user profile.
+
+| Name                       | Type | Description                | Unique | Required | Editable | Searchable |
+| -------------------------- | ---- | -------------------------- | ------ | -------- | -------- | ---------- |
+| password_encrypted         | text | Encrypted password         | ❌     | ❌       | ✅       | ❌         |
+| password_encryption_method | text | Password encryption method | ❌     | ❌       | ✅       | ❌         |
+
+Logto uses [Argon2](https://en.wikipedia.org/wiki/Argon2)
+hashing algorithm and its implementation [node-argon2](https://github.com/ranisalt/node-argon2)
+to secure the user password by default.
```

Comment:
Could have a separate section for passwords introduction.

Reply:
removed this introduction here for now.
will add it back in the "Basic data" section.